docker-compose.dev.yml

```diff
@@ -67,6 +67,22 @@ services:
         socks_proxy:
       dockerfile: Dockerfile.ui
 
+  cvat_redis:
+    ports:
+      - '6379:6379'
```

Comment:
`docker-compose -f docker-compose.yml -f docker-compose.dev.yml   up -d --build` doesn't work for me anymore because I have redis on my computer. What is a reason to export the port?

Reply:
OK. Probably it is fine. But I don't like that in the dev file we open so many ports. It can lead to conflicts.
I have started thinking that we need analytics.yml for all new services.

Reply:
I will recommend not opening the port, because if a user has running redis on the host, it will lead to problems.

Reply:
removed